baipw/utils.py

```diff
@@ -7,26 +7,12 @@
 
 
 def get_client_ip(request):
-    # IP retrieved from CloudFlare
-    cf_connecting_ip = request.META.get("HTTP_CF_CONNECTING_IP")
-
-    # Header usually set by proxies
-    x_forwarded_for = request.META.get("HTTP_X_FORWARDED_FOR")
-
-    # Header set by the connecting party, usually not the actual client making
-    # the request, but a web server that the request goes through.
-    remote_addr = request.META.get("REMOTE_ADDR")
-
-    # Prioritise IPs from proxies.
-    final_ip = cf_connecting_ip or x_forwarded_for or remote_addr
-
-    # If no IP address was attached to the address, return nothing.
-    if final_ip is None:
-        return
+    """
+    Get the client's IP address
 
-    # If there is a list of IPs provided, use the last one (should be
-    # the most recent one). This may not work on Google Cloud.
-    return final_ip.split(",")[-1].strip()
+    Note: This is the address connecting to Django, which is likely incorrect.
+    """
+    return request.META.get("REMOTE_ADDR")
```

Comment:
Given this can return `None` technically, should we make sure to add a None check in the `_get_client_ip` method?

Reply:
I'd always assumed `REMOTE_ADDR` would always exist, but according to Django it's not a 100% guarantee, so I agree I'll add some tests!.